tests/tt_metal/tt_metal/perf_microbenchmark/1_compute_mm/test_compute_mm.cpp

```diff
@@ -243,58 +376,167 @@ int main(int argc, char** argv) {
+
+            } else {
+
+                // auto temp = CreateBuffer(tt::tt_metal::ShardedBufferConfig{
+                //                                     .device = device,
+                //                                     .size = 1024*512*2,
+                //                                     .page_size = 2048,
+                //                                     .buffer_layout = TensorMemoryLayout::HEIGHT_SHARDED,
+                //                                     .shard_parameters = ShardSpecBuffer(
+                //                                         CoreRangeSet(std::set<CoreRange>({ CoreRange(CoreCoord(0,0))})),
+                //                                         {1024, 512},
+                //                                         ShardOrientation::ROW_MAJOR,
+                //                                         false,
+                //                                         {tt::constants::TILE_HEIGHT, tt::constants::TILE_WIDTH},
+                //                                         {32, 16})});
+
+                auto activations_tilized = tilize(tensor_in0_fp8.get_values(), M, K);
+                std::vector<uint32_t> activations = pack_fp32_vec_as_bfp8_tiles(activations_tilized, true, false);
+                input_buffer0 = create_and_transfer_data_sharded_cb_fp8(device, activations, Mt, Kt);
+
+                // tt::Cluster::instance().l1_barrier(device->id());
+                // std::vector<uint32_t> output;
+                // tt::tt_metal::detail::ReadFromBuffer(input_buffer0, output);
+                // pass = (output == activations);
+
+                // in1
+                auto identity_tilized = tilize(tensor_in1_fp8.get_values(), K, N);
+                auto weights = pack_fp32_vec_as_bfp8_tiles(identity_tilized, true, false);
+                input_buffer1 = create_and_transfer_data_sharded_cb_fp8(device, weights, Kt, Nt);
+
+                // output
+                SHAPE output_hsape = {1, 1, M, N};
+                tt::deprecated::Tensor<float> out_tensor = tt::deprecated::initialize_tensor<float>(output_hsape, tt::deprecated::Initialize::ZEROS, 100, std::chrono::system_clock::now().time_since_epoch().count());;
+                auto output_tilized = tilize(out_tensor.get_values(), M, N);
+                auto outputs = pack_fp32_vec_as_bfp8_tiles(output_tilized, true, false);
+                output_buffer = create_and_transfer_data_sharded_cb_fp8(device, outputs, Mt, Nt);
+            }
+        }
+
         ////////////////////////////////////////////////////////////////////////////
         //                      Application Setup
         ////////////////////////////////////////////////////////////////////////////
         auto [math_fidelity, fp32_dest_acc_en] = get_compute_params(arch);
+        math_fidelity = fidel == 0 ? MathFidelity::LoFi : MathFidelity::HiFi2;
```

Comment:
The current code sets the math fidelity for GS to HiFi4 and HiFi2 for WH_B0.
From now on, will matmul benchmarks only use LoFi and HiFi2?

Reply:
I will change this back for the multi-core version